index.d.ts

```diff
@@ -0,0 +1,611 @@
+/* tslint:disable:interface-name max-classes-per-file no-empty-interface */
+
+declare module 'evergreen-ui' {
+  import * as React from 'react'
```

Comment:
Can this be `import React from 'react'`?

Reply:
Nope.
TS1259: Module '"./evergreen/node_modules/@types/react/index"' can only be default-imported using the 'esModuleInterop' flag